vhost_user_fs/src/fs_cache_req_handler.rs

```diff
@@ -0,0 +1,61 @@
+use crate::fuse;
+use std::io;
+use std::os::unix::io::RawFd;
+use vhost_rs::vhost_user::message::{
+    VhostUserFSSlaveMsg, VhostUserFSSlaveMsgFlags, VHOST_USER_FS_SLAVE_ENTRIES,
+};
+use vhost_rs::vhost_user::{SlaveFsCacheReq, VhostUserMasterReqHandler};
+
+/// Trait for virtio-fs cache requests operations.  This is mainly used to hide
+/// vhost-user details from virtio-fs's fuse part.
+pub trait FsCacheReqHandler: Send + Sync + 'static {
+    /// Setupmapping operation
+    fn map(
+        &mut self,
+        foffset: u64,
+        moffset: u64,
+        len: u64,
+        flags: u64,
+        fd: RawFd,
+    ) -> io::Result<()>;
+
+    /// Removemapping operation
```

Comment:
Or this one.

Reply:
`Remove those mappings that provide the access to file data.`
--
how about this?